huxley/www/static/js/huxley/components/RegistrationView.js

```diff
@@ -367,6 +367,8 @@ var RegistrationView = React.createClass({
               <li>
                 <input
                   type="text"
+                  id="input_delegates"
```

Comment:
What do this `id` and `name` attributes do? They don't seem to be referenced anywhere else in this diff.